layers/+lang/python/packages.el

```diff
@@ -407,3 +408,20 @@ fix this issue."
       (when python-enable-yapf-format-on-save
         (add-hook 'python-mode-hook 'yapf-mode)))
     :config (spacemacs|hide-lighter yapf-mode)))
+
+;;https://github.com/andrew-christianson/lsp-python-ms
+;;Not on MELPA
+(defun python/init-lsp-python-ms ()
+  (use-package lsp-python-ms
+    :if (eq python-backend 'lsp-ms)
+    :ensure nil
+    :config
+
+    (if python-lsp-git-root
+      ;; Use dev version of language server checked out from github
+      (progn
+        (setq lsp-python-ms-dir
+          (expand-file-name (concat python-lsp-git-root "/output/bin/Release/")))
+        (message ( concat "lsp-python-ms: Using version at " lsp-python-ms-dir)))
```

Comment:
more common is to use %s with message instead of concat.

Reply:
Thanks @yyoncho -- amended/pushed